analyzer/build.gradle

```diff
@@ -35,6 +37,14 @@ version = '0.3.2'
 sourceCompatibility = JavaVersion.VERSION_1_6
 targetCompatibility = JavaVersion.VERSION_1_6
 
+sourceSets {
+    main.java.srcDir 'src/main/build'
+}
```

Comment:
I just removed lines 40 to 46. Seems to work for me locally. 
What's the issues you're experiencing?